config/Zed/config.dist.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Copy over the following configs to your config
+ */
+
+use SprykerEco\Shared\Computop\ComputopConstants;
+
+$config[ComputopConstants::COMPUTOP_MERCHANT_ID] = 'COMPUTOP_MERCHANT_ID'; //Set up real data
+$config[ComputopConstants::COMPUTOP_BLOWFISH_PASSWORD] = 'COMPUTOP_BLOWFISH_PASSWORD'; //Set up real data
+$config[ComputopConstants::COMPUTOP_HMAC_PASSWORD] = 'COMPUTOP_HMAC_PASSWORD'; //Set up real data
+
+$config[ComputopConstants::COMPUTOP_CREDIT_CARD_ORDER_ACTION] = 'https://www.computop-paygate.com/payssl.aspx';
+$config[ComputopConstants::COMPUTOP_CREDIT_CARD_AUTHORIZE_ACTION] = 'https://www.computop-paygate.com/authorize.aspx';
+$config[ComputopConstants::COMPUTOP_CREDIT_CARD_CAPTURE_ACTION] = 'https://www.computop-paygate.com/capture.aspx';
+$config[ComputopConstants::COMPUTOP_CREDIT_CARD_REVERSE_ACTION] = 'https://www.computop-paygate.com/reverse.aspx';
+$config[ComputopConstants::COMPUTOP_CREDIT_CARD_INQUIRE_ACTION] = 'https://www.computop-paygate.com/inquire.aspx';
+$config[ComputopConstants::COMPUTOP_CREDIT_CARD_REFUND_ACTION] = 'https://www.computop-paygate.com/credit.aspx';
+
+$config[ComputopConstants::COMPUTOP_TEST_ENABLED] = true; //Test mode enabled
+$config[ComputopConstants::COMPUTOP_REFUND_SHIPMENT_PRICE_ENABLED] = true; //Refund with shipment price
+
+$config[ComputopConstants::COMPUTOP_RESPONSE_MAC_REQUIRED] = [
```

Comment:
Those need to go away as env config and should only be config constants for Config class. No env leaking :)

Reply:
Updated. Please check.